packages/timeline-state-resolver/src/integrations/vmix/index.ts

```diff
@@ -793,6 +794,27 @@ export class VMixDevice extends DeviceWithState<VMixStateExtended, DeviceOptions
 					timelineId: '',
 				})
 			}
+			if (input.listFilePaths !== undefined && !_.isEqual(oldInput.listFilePaths, input.listFilePaths)) {
+				commands.push({
```

Comment:
will this result in other properties of the list being reset too?
what happens if a clip from the list is playing when this is run?

Reply:
> will this result in other properties of the list being reset too?

No, other properties such as "sharpen" and "mirror" are retained.

> what happens if a clip from the list is playing when this is run?

It too is removed.

Reply:
> > what happens if a clip from the list is playing when this is run?
> 
> It too is removed.

Won't that be problematic?  
When playing a clip, and a new one is added to the list by a lookahead (or whatever causes it to be added to the list). Which results in the playing clip to be removed and re-added?

I assume it is the case that there is a list for each AB player. So maybe this scenario couldn't be hit, but I would want to have the next clip preloaded while the current one is playing to allow for seamless playback, or user confidence that the clip does load.

Reply:
I think this is fine for now. I suppose there could be an option of individually removing the _removed_ items from that array, and then adding the ones added, but personally, I feel this could just be fixed by whoever needs it fixed.

Reply:
@Julusian are you satisfied with leaving this for now, or do you see a scenario where lookaheads _can_ wreak havoc here?